interactive_shell_handler.c

```diff
@@ -0,0 +1,150 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <windows.h>
+#include <psapi.h>
+#include <tchar.h>
+#include "interactive_shell_handler.h"
+#include "utils.h"
+
+// forward declarations
+void perform_command(char *command);
+char** split_command(char *command);
+
+void handle_copy(char* src, char* dst) {
+    int exitCode = 0;
+
+    exitCode = copy_file(src, dst);
+    if (exitCode == 0) {
+        printf("copied source file to destination");
+    } else {
+        printf("failed to copy source file to destination");
+    }
+}
+
+void handle_move(char* src, char* dst) {
+    int exitCode = copy_file(src, dst);
+    if (exitCode == 0) {
+        // delete src file 
+        exitCode = remove(src);
+        if (exitCode == 0) {
+            printf("moved file successfully");
+        } else {
+            printf("failed to moved file");
+        }
+    }
+}
+
+void print_process_name_and_id (DWORD processId) {
+    TCHAR processName[MAX_PATH] = "<unknown>";
```

Comment:
I like the trick here, but you should probably propagate the underlying error to the user somehow (why wasn't the process found? Access denied? Something else?)